test/lsp-int-test-project/definitions.mo

```diff
@@ -1,9 +1,11 @@
 import MyDependency "mo:mydep/lib";
 import List "lib/list";
+import { last } "lib/list";
```

Comment:
I suggest to also add a _renaming on import_ declaration.

Reply:
Thanks for taking a look. I added an LSP test based on your suggestion (slightly modified since there seems to be a compiler bug resulting in `type error [M0121], duplicate field last in object pattern`). I created an issue #3290 and will see if there's an elegant way to fix this in a separate PR.

Reply:
Nice find!